tasks/parse-pdr/tests/parse_pdrs_test.js

```diff
@@ -46,6 +46,26 @@ test.beforeEach(async (t) => {
   await t.context.collectionConfigStore.put('MOD09GQ', '006', collectionConfig);
 });
 
+const testGranule = {
+  granuleId: 'MOD09GQ.A2017224.h09v02.006.2017227165020',
```

Comment:
The actual change to this file was to add fileType to the tests coming back from parse PDR.  The test refactor dried up some of the repetition in that to make updates a bit less copypasta-ish.